openbb_sdk/providers/fmp/openbb_fmp/models/market_snapshots.py

```diff
@@ -0,0 +1,160 @@
+"""FMP Market Snapshots fetcher."""
+
+from typing import Any, Dict, List, Literal, Optional
+
+import pandas as pd
+from openbb_provider.abstract.fetcher import Fetcher
+from openbb_provider.standard_models.market_snapshots import (
+    MarketSnapshotsData,
+    MarketSnapshotsQueryParams,
+)
+from openbb_provider.utils.helpers import make_request
+from pydantic import Field
+
+MARKETS = Literal[
```

Comment:
`MARKETS` belongs to `openbb_fmp.utils.helpers` file. This reduces congestion in the code.

Reply:
Will move this.